libpod/options.go

```diff
@@ -1692,6 +1692,14 @@ func WithSecrets(secretNames []string) CtrCreateOption {
 	}
 }
 
+// WithPidFile adds pidFile to the container
+func WithPidFile(pidFile string) CtrCreateOption {
+	return func(ctr *Container) error {
+		ctr.config.PidFile = pidFile
```

Comment:
Needs a check for `ctr.valid` first - see other container options

Reply:
Sorry for missing it. Have added